gcp/src/ARCFileReader.cxx

```diff
@@ -108,13 +104,23 @@ class ARCFileReader : public G3Module {
 	std::deque<std::string> filename_;
 	std::string cur_file_;
 
+	int experiment;
```

Comment:
There seems to be a type mismatch here: int in the member variable, unsigned in the constructor, and enum for the definition.